src/documents/yesNo.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Yes/No
+layout: default
+category: API
+---
+
+## Overview
+
+**Component Name:** `gpii.firstDiscovery.panel.yesNo`
+
+**File:** `panels.js`
+
+Provides a means to create panels that select and only select one of two choices, such as yes or no.
+This component is not intended to be used on its own, but provided as a base grade to another
+component that will use it's capabilities for creating such panels.
```

Comment:
"... will use it's... " should be "... will use its...". Remove the single quote.